modules/invibesBidAdapter.js

```diff
@@ -644,6 +658,57 @@ invibes.getCookie = function (name) {
   return storage.getCookie(name);
 };
 
+invibes.setCookie = function(name, value, exdays, domain) {
+  if (!storage.cookiesAreEnabled()) {
+    return;
+  }
+
+  if (!invibes.optIn || !invibes.purposes[0]) {
+    return;
+  }
+
+  if (exdays > 365) {
+    exdays = 365;
+  }
+
+  domain = domain || detectTopmostCookieDomain();
+  let exdate = new Date();
+  let exms = exdays * 24 * 60 * 60 * 1000;
+  exdate.setTime(exdate.getTime() + exms);
+
+  storage.setCookie(name, value, exdays ? exdate.toUTCString() : '', null, domain);
```

Comment:
why would you set a cookie? Publishers hate this. Why not use local storage?

Reply:
Hey Patrick,

We are first checking to see if we have consent and then on our servers we also decode and read the TC String. Only when we have full consent we are setting cookies. 
This cookie that we are setting, is something that the publishers we work with are aware of, as it helps with user sync. Please let me know if you have any other questions. Also, we are already setting this cookie right now when we are winning the bid, so this will not affect any relationship.

Reply:
Hey Patrick, please do NOT accept yet this Pull Request as we found a bug on our side. I will fix it and commit as soon as possible.

Reply:
This behavior is quite frowned upon. Prebid recently allowed publisher to block it and will block it by default imminently. Why not use local storage?

Reply:
Hey Patrick,

Can you please tell me if this setting that will be off by default will it extend to reading cookies as well?

Reply:
Change default of https://github.com/prebid/Prebid.js/issues/7280 for bidders; bid modules will not be able to use storage manager unless whitelisted by pubs in config is included in #7796

Reply:
Thank you for the informations. 
I will work on updating the Invibes bidder to user localStorage instead of cookies. I'll push the updates to this Pull Request as soon as we have it implemented and tested on our environment.

Reply:
Hey @patmmccann ,

We have made the necessary changes and tested them thoroughly on our side. I have only one issue, it looks like the some automatic tests are failing, but on someone else's adapter "FeedAdAdapter". 

We are not allowed to push code changes for others. Can you help me?  
Thank you

Reply:
Hey @jsnellbaker  @patmmccann ,

Can you please provide me an update? Like I mentioned before, the automated tests seems to be failing with someone else's adapter.

Thank you!

Reply:
Hi @mihaisandu07 ,
Generally, if the test is failing for someone else, you can ignore it.  We'll take that into account when doing the overall review for your code changes.

Reply:
@mihaisandu07 please submit a docs pull with a disclosure of this behavior, thanks!

Reply:
Hey Patrick, 

The code that you've quoted no longer exists. The invibes prebid adapter doesn't write any cookies now. Are you referring to something else?

Reply:
Hey @patmmccann, let me know if I missed something, but in the last code update we don't have anything that writes in cookies. Is that message in regards to something else?